### PR TITLE
Add empty states and other design improvements

### PR DESCRIPTION
**Pull request by maciaszczykm**
## Summary
Closes ENG-382.

[Design](https://www.figma.com/file/ThREg5Q4RMkKt5haMwQ4Ar/Notifications%2C-Updates%2C-Empty-States?node-id=14%3A3962).

- Add empty state component
- Add empty states for:
  - Clusters
  - Marketplace installations
  - Audits
  - Repository readme, tests and deployments
  - Account groups, roles, service accounts and domains
  - Profile access tokens, public keys and EAB credentials
- Redesign audits to use the new layout, add some missing paddings, page titles etc.
- Adjust space between chips in packages (closes ENG-460)
- Fix scrollbar on repo readmes (closes ENG-461)
- Fix Docker pull codeline (fixes ENG-403)

Zrzut ekranu 2022-08-4 o 15 01 55
Zrzut ekranu 2022-08-4 o 15 29 02
Zrzut ekranu 2022-08-4 o 13 44 46
Zrzut ekranu 2022-08-4 o 13 44 30

## Test Plan
Tested manually.

## Checklist
- [x] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [x] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: www/src/components/account/Groups.js [hunk 1]
```diff
@@ -154,29 +156,35 @@ function GroupsInner({ q }) {
...
+          loadNextPage={() => pageInfo.hasNextPage && fetchMore({
+            variables: { cursor: pageInfo.endCursor },
+            updateQuery: (prev, { fetchMoreResult: { groups } }) => extendConnection(prev, groups, 'groups'),
+          })}
+          hasNextPage={pageInfo.hasNextPage}
+          loading={loading}
+          placeholder={Placeholder}
+        />
+      ) : (
+        <EmptyState message={isEmpty(q) ? "Looks like you don't have any groups yet." : "Looks like you don't have any groups matching search criteria."}>
```
Comment (michaeljguarino) [inline www/src/components/account/Groups.js:184]: change to `No groups found for {q}`

File: www/src/components/account/Roles.js [hunk 2]
```diff
@@ -117,29 +119,35 @@ function RolesInner({ q }) {
...
+          loadNextPage={() => pageInfo.hasNextPage && fetchMore({
+            variables: { cursor: pageInfo.endCursor },
+            updateQuery: (prev, { fetchMoreResult: { roles } }) => extendConnection(prev, roles, 'roles'),
+          })}
+          hasNextPage={pageInfo.hasNextPage}
+          loading={loading}
+          placeholder={Placeholder}
+        />
+      ) : (
+        <EmptyState message={isEmpty(q) ? "Looks like you don't have any roles yet." : "Looks like you don't have any roles matching search criteria."}>
```
Comment (michaeljguarino) [inline www/src/components/account/Roles.js:147]: Same as the groups search text

File: www/src/components/account/Domains.js [hunk 3]
```diff
@@ -281,57 +281,60 @@ export function Domains() {
...
+
+            <StandardScroller
+              listRef={listRef}
+              setListRef={setListRef}
+              hasNextPage={pageInfo.hasNextPage}
+              items={edges}
+              loading={loading}
+              placeholder={Placeholder}
+              mapper={({ node }, { next }) => (
+                <TableRow
```
Comment (michaeljguarino) [inline www/src/components/account/Domains.js:303]: We should probably refactor this role into a separate function component tbh

**Comment by maciaszczykm** [comment]
@michaeljguarino Fixed, PTAL.

**Review by michaeljguarino — APPROVED** [review]

**Pull request merged**